### PR TITLE
desktop: enable pipewire by default

### PR DESCRIPTION
**Pull request by zimbatm**

### CODE REVIEW

File: desktop/pipewire.nix [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Use pipewire instead of PuleseAudio. See https://nixos.wiki/wiki/PipeWire
+{
+  security.rtkit.enable = true;
+  services.pipewire = {
+    enable = true;
+    alsa.enable = true;
+    alsa.support32Bit = true;
```
Comment (Mic92) [inline desktop/pipewire.nix:7]: Probably does not make sense on other architectures than x86:

```suggestion
    alsa.support32Bit = pkgs.stdenv.isx86_64;
```

File: desktop/pipewire.nix [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Use pipewire instead of PuleseAudio. See https://nixos.wiki/wiki/PipeWire
+{
+  security.rtkit.enable = true;
```
Comment (Mic92) [inline desktop/pipewire.nix:3]:
```suggestion
  security.rtkit.enable = lib.mkDefault true;
```

File: desktop/pipewire.nix [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  services.pipewire = {
+    enable = true;
```
Comment (Mic92) [inline desktop/pipewire.nix:5]:
```suggestion
    enable = lib.mkDefault true;
```

File: desktop/pipewire.nix [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+    alsa.enable = true;
```
Comment (Mic92) [inline desktop/pipewire.nix:6]:
```suggestion
    alsa.enable = lib.mkDefault true;
```

File: desktop/pipewire.nix [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+    alsa.support32Bit = true;
+    pulse.enable = true;
```
Comment (Mic92) [inline desktop/pipewire.nix:8]:
```suggestion
    pulse.enable = lib.mkDefault true;
```

File: desktop/pipewire.nix (hunk 1 above)
Comment (Mic92) [reply desktop/pipewire.nix:7]: But also, what applications are still 32-bit? I have not any in a while.

Comment (zimbatm) [reply desktop/pipewire.nix:7]: Good call. I added this for Steam and could be enabled separately.

Comment (Mic92) [reply desktop/pipewire.nix]: I see. I guess with games we will have 32-bit at least for another decade.

**Comment by zimbatm** [comment]
bors merge

**Comment by Mic92** [comment]
We need to whitelist bors...

**Comment by zimbatm** [comment]
bors merge

**Pull request merged**